ajax/libs/phery.js/package.json

```diff
@@ -0,0 +1,43 @@
+{  
+   "name": "phery.js",
+   "filename": "phery.min.js",
+   "description": "The bridge from jQuery to PHP, and vice-versa through AJAX.",
+   "license": "MIT",
+   "keywords": [  
+      "AJAX",
+      "event",
+      "unobtrusive",
+      "fileupload",
+      "PHP",
+      "form",
+      "serialization",
+      "partial",
+      "REST",
+      "upload",
+      "FormData",
+      "jquery",
+      "dom",
+      "HTML5",
+      "JSON"
+   ],
+   "repository": {  
+      "type": "git",
+      "url": "git://github.com/pheryjs/phery.git"
+   },
+   "homepage": "http://phery-php-ajax.net",
+   "author": {  
+      "name": "Paulo Cesar",
+      "url": "http://github.com/pocesar"
+   },
+   "dependencies": {  
+      "jquery": ">=1.8"
+   },
+   "autoupdate": {  
+      "source": "git",
+      "target": "git://github.com/pheryjs/phery.git",
+      "basePath": "",
+      "files": [  
+         "phery*.+(js|css|map)"
+      ]
+   }
+}
```

Comment:
mm, ok, true. 

I used https://jsonformatter.curiousconcept.com/ to format it, maybe you guys can suggest/include a formatter to use (which also uses spaces after the double colon: ```key: "value"``` instead of ```key:"value"```..